docs/vanilla.md

```diff
@@ -305,4 +306,45 @@ Download CRD has some key phases for the `.status.phase` field:
 * InProgress: download is in progress
 * Completed: download is completed
 * Retry: download is retried. When there is any failure during the download of backup data, download will be retried
-* Failed: download is failed
\ No newline at end of file
+* Failed: download is failed
+
+### StorageClass Mapping
```

Comment:
make this a standalone doc

Reply:
Agree with Xing. Please keep this as a standalone doc. 

StorageClass Mapping is an option of restore, so it is not good to create a new section that is parallel with the `Restore` section. Instead, you can add a reference before the following line in vanilla.md.
```
For more restore options, please refer to [Velero Document](https://velero.io/docs/v1.5/).
```
For example,
```
For the volume restores with changing storage class, please refer to <the link to your storage class mapping doc>
```

Reply:
Moved storage class mapping part to a standalone doc.